app/helpers/feed_helper.rb

```diff
@@ -13,7 +19,7 @@ def documents_as_feed_entries(documents, builder, feed_updated_timestamp = Time.
     builder.updated feed_updated_timestamp
 
     documents.each do |document|
-      builder.entry(document, id: document_id(document, builder), url: public_document_url(document), published: document.first_public_at, updated: document.public_timestamp) do |_entry|
+      builder.entry(document, id: document_id(document, builder), url: public_document_url(document), published: document.try(:first_public_at), updated: document.public_timestamp) do |_entry|
```

Comment:
Given that `first_public_at` doesn't appear to be a field in the Search API <sup>[1]</sup> could we either:

- fall back to `first_published_at` from the Search API _or_
- alias `first_public_at` to `first_published_at` in our models

[1]: https://docs.publishing.service.gov.uk/apis/search/fields.html

Reply:
Neither first_published_at or first_public_at are reliably provided, so the option I went for was to default to not providing a `published` field in the feeds for rummager documents.

Do you think we should try getting `first_published_at`? I wasn't able to find any docs with this field.